package.json

```diff
@@ -367,7 +367,7 @@
             "null"
           ],
           "default": null,
-          "description": "Specifies the OmniSharp to use. Can be one of \"latest\", a specific version number, or the absolute path to a local OmniSharp folder."
+          "description": "Specifies how to acquire the OmniSharp to use. Can be one of \"latest\", a specific version number, or the absolute path to a local OmniSharp folder."
```

Comment:
Still seems awkward...I think the problem is the phrase "the Omnisharp". Maybe "The copy of Omnisharp"? @DustinCampbell ?

Reply:
I think it might be easier to be plain and then add a bit more explaining text at the end. Also, I believe that "the absolute path to a local OmniSharp folder" is actually incorrect. IIRC, the path needs to point to the file that will be launched.

Here's a stab a some new text. Additional word-smithing is welcome!

"Specifies the path to OmniSharp. This can be the absolute path to an OmniSharp executable, a specific version number, or \"latest\". If a version number or \"latest\" is specified, the appropriate version of OmniSharp will be downloaded on your behalf."

Reply:
Do we need to specify that "latest" implies the latest CI build, because just the word "latest" might also imply the latest released version to the user ?